packages/cli/test/functional/test_site/expected/requirements/NonFunctionalRequirements._include_.html

```diff
@@ -3,9 +3,7 @@
     <p>In contrast to <strong>Functional Requirements</strong> which specify what the system should do,
       <strong>Non-Functional Requirements</strong> specify the constraints under which system is developed and operated.</p>
   </div>
-  <p>
-    <seg id="preview">Requirements gathering, requirements elicitation, requirements analysis,
-      requirements capture are some of the terms commonly <strong>and</strong> interchangeably used to represent the activity
-      of understanding what a software product should do.</seg>
-  </p>
+  <seg id="preview">Requirements gathering, requirements elicitation, requirements analysis,
```

Comment:
should we change this to an inline tag (e.g. `<span>`?) to preserve the `<strong>` tag?

Reply:
Hmm yep, that would be good. On another note, should we update `<seg>` to something else in the SSR PR since we don't "allow" unknown components?